SkyDrive/SkyDrive.Tests/UnitTests/EventServiceTests.cs

```diff
@@ -0,0 +1,266 @@
+            _mockRepository.Setup(x => x.Update(It.IsAny<EventEntity>(), CancellationToken.None)).ReturnsAsync(eventEntity);
+            var service = new EventService(_mockRepository.Object);
+
+            //Act
+            var result = await service.UpdateEvent(eventModel, CancellationToken.None);
+
+            //Assert
+            _mockRepository.Verify(x => x.GetById(3, CancellationToken.None), Times.Once);
+            _mockRepository.Verify(x => x.Update(It.IsAny<EventEntity>(), CancellationToken.None), Times.Once);
+            Assert.NotNull(result);
+            Assert.IsType<EventModel>(result);
+        }
+
+        [Fact]
+        public async Task UpdateEvent_IncorrectModel_ReturnEntityNotFoundException()
+        {
+            //Arrange
+            var eventModel = new EventModel
+            {
+                Id = 3,
+                DateTimeOfEvent = DateTime.Now.AddDays(1),
+                InstructorId = 2,
+                Instructor = new InstructorModel(),
+                Members = new List<MemberModel>()
+            };
+
+            _mockRepository.Setup(x => x.GetById(3, CancellationToken.None)).ReturnsAsync(value: null);
+            var service = new EventService(_mockRepository.Object);
+
+            //Act
+            var result = await Assert.ThrowsAsync<EntityNotFoundException>(() => service.UpdateEvent(eventModel, CancellationToken.None));
+
+            //Assert
+            _mockRepository.Verify(x => x.GetById(3, CancellationToken.None), Times.Once);
+            Assert.NotNull(result);
+            Assert.Equal("Event with id: 3 not found", result.Message);
+        }
+
+        [Fact]
+        public async Task DeleteEvent_CorrectId_ReturnNull()
+        {
+            //Arrange
+            var eventEntity = new EventEntity
+            {
+                Id = 3,
+                DateTimeOfEvent = DateTime.Now.AddDays(1),
+                InstructorId = 2,
+                Instructor = new InstructorEntity(),
+                Members = new List<MemberEntity>()
+            };
+
+            _mockRepository.Setup(x => x.GetById(3, CancellationToken.None)).ReturnsAsync(eventEntity);
+            _mockRepository.Setup(x => x.Delete(It.IsAny<EventEntity>(), CancellationToken.None));
+            var service = new EventService(_mockRepository.Object);
+
+            //Act
+            await service.DeleteEvent(3, CancellationToken.None);
+
+            //Assert
+            _mockRepository.Verify(x => x.GetById(3, CancellationToken.None), Times.Once);
+            _mockRepository.Verify(x => x.Delete(It.IsAny<EventEntity>(), CancellationToken.None), Times.Once);
+        }
+
+        [Fact]
+        public async Task DeleteEvent_IncorrectId_ReturnArgumentNullException()
+        {
+            //Arrange
+            _mockRepository.Setup(x => x.GetById(3, CancellationToken.None)).ReturnsAsync(value: null);
```

Comment:
what is '3'? it's not obvious. probably, move it to separate variable with proper name